source/lib-intro.tex

```diff
@@ -3340,8 +3340,8 @@
 \end{itemize}
 
 \pnum
-Calling the \tcode{set_*} and \tcode{get_*} functions shall not incur a data race. A call to
-any of the \tcode{set_*} functions shall synchronize with subsequent calls to the same
+Calling the \tcode{set_*} and \tcode{get_*} functions shall not incur a data race\iref{intro.races}.
+A call to any of the \tcode{set_*} functions shall synchronize with subsequent calls to the same
```

Comment:
The additional xref is an improvement, but "shall synchronize" should just be "synchronizes"
and `get_*` isn't really the name of a function (shell globs are not a defined thing in the C++ standard).

Reply:
Fixed by https://github.com/cplusplus/draft/pull/6850.